bokeh/embed.py

```diff
@@ -34,15 +42,19 @@ def components(plot_object, resources):
     Args:
         plot_object (PlotObject) : Bokeh object to render
             typically a Plot or PlotContext
-        resources (Resources, optional) : BokehJS resources config
-
+        resources : Deprecated argument
     Returns:
         (script, div) : UTF-8 encoded
 
     '''
+    
+    if resources is not None:
+        warn('the ``resources`` argument is deprecated and will be removed in'
+             'a future version.', DeprecationWarning, stacklevel=2)
+    
```

Comment:
That's fine. We have decorators when an entire function is deprecated, but for cases like this we just issue the `Deprecation Warning`. However we need to add more info to the message, we always try to point users at whatever the replacement is for the thing being deprecated. In this instance we should explain that the parameter is just no longer needed at all.